### ISSUE TITLE
OMMatlab: Documentation buggy

### ISSUE THREAD

**Issue by AHaumer** [OpenModelica/OpenModelica#12688]
Have a look at https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/ommatlab.html:
![grafik](https://github.com/OpenModelica/OpenModelica/assets/6900273/583ba92e-c4d7-4108-9580-3fb407ec7ef5)

1. Isn't prefix "omc." mssing in the second and third command just before "ModelicaSystem"?
2. I suppose the "mod" in the second and third command are wrong and should be left away?
3. If the package is stored in a directory structure (not as one file), how can it be opened?

![grafik](https://github.com/OpenModelica/OpenModelica/assets/6900273/55ed3d7d-90fe-446c-83e4-789bf996ef9d)
From where do I get the "mod" in this command?

**Comment by casella** [comment]
@arun3688 can you please take care of this when you'll have some time?

**Comment by arun3688** [comment]
@casella sure i will look into it as early as possible

**Comment by arun3688** [comment]
@AHaumer you are correct the `mod` should be replaced to omc prefix object, it was a typo I will fix it,

> 3\. If the package is stored in a directory structure (not as one file), how can it be opened?

Let us assume that your package is stored in a directory with top level package.mo in the following path `C:/powersystem/package.mo` and the model you want to simulate is test  then you can load it the following way

```
import OMMatlab.*
omc= OMMatlab()
omc.ModelicaSystem("C:/powersystem/package.mo", "powersystem.test")
omc.simulate()
omc.getSolutions()
```

**Comment by casella** [comment]
This was actually fixed in July 2024, closing.
